### ISSUE TITLE
Add information on creating Read-Only filesystems

### ISSUE THREAD

**Issue by aallan** [raspberrypi/documentation#3266]
How read only filesystems work has changed in Bookworm. See raspberrypi/bookworm-feedback#137 for details. More details need adding to the documentation. There is already something in raspi-config about the overlay filesystem,

![image](https://github.com/raspberrypi/bookworm-feedback/assets/53625/c8d87447-3fb0-4248-b1fd-3c70ea46900

but this option doesn't necessarily do what some end users would expect. Should add more information to [this section](https://www.raspberrypi.com/documentation/computers/configuration.html#overlay-file-system) I guess?

**Comment by lurch** [comment]
It's not only in raspi-config, it's also in the Raspberry Pi Configuration GUI tool.

**Comment by aallan** [comment]
Too soon Github bot, too soon.

**Comment by theepicflyer** [comment]
Agree! I just spent an afternoon figuring out how to set overlayroot to overlay just root and not /home. I have a separate /home partition and want to keep that writable for user data.

Mainly I was misled by this project on github also called [overlayroot](https://github.com/chesty/overlayroot). I thought this was overlayroot and tried to customize it this way. The Debian [cloud-initramfs-tools](https://salsa.debian.org/cloud-team/cloud-initramfs-tools) does not come up in a Google search, and I only found it after I went through the Debian package browser.

Please add documentation to customize overlayroot!

**Comment by nathan-contino** [comment]
Still important. Just need more resources to get to it.

**Comment by stevedavies-rp** [comment]
Information added by @JeuneseRPi

**Comment by katshann-raspberry** [comment]
https://www.raspberrypi.com/documentation/computers/configuration.html#overlay-filesystem
